public/app/core/services/echo/backends/analytics/GA4Backend.ts

```diff
@@ -41,9 +47,10 @@ export class GA4EchoBackend implements EchoBackend<PageviewEchoEvent, GA4EchoBac
     if (!window.gtag) {
       return;
     }
-
-    window.gtag('set', 'page_path', e.payload.page);
-    window.gtag('event', 'page_view');
+    // this should prevent duplicate events in case enhanced tracking is enabled
+    if (!this.googleAnalytics4SendManualPageViews) {
+      window.gtag('event', 'page_view', { page_path: e.payload.page });
+    }
```

Comment:
should this conditional be inverted? `if (sendPageViews) { sendPageView() }` ?

Reply:
Changed https://github.com/grafana/grafana/pull/57619/commits/4963aaaa72a57478eba5d1d6f01c232fafbec4cd